pages/common/enscript.md

```diff
@@ -0,0 +1,23 @@
+# enscript
+
+> A tool to convert text files to PostScript, HTML, RTF, ANSI, and overstrikes.
+
+- Generate PostScript from a file and output to another:
+
+`enscript {{path/to/input_file}} --output={{path/to/output_file}}`
+
+- Generate a certain output language (eg. _html_) from a file and output to another:
+
+`enscript {{path/to/input_file}} --language={{language}} --output={{path/to/output_file}}`
+
+- Generate PostScript from a file and output to another with 1 to 9 column per page in landscape:
+
+`enscript {{path/to/input_file}} --columns={{num}} --landscape --output={{path/to/output_file}}`
+
+- Display avaliable syntax highlighting:
+
+`enscript --help-highlight`
+
+- Genereate PostScript from a file and output to another with syntax highlighting and color for a specified language:
```

Comment:
typo in Generate

Reply:
I deleted my fork, sorry. I'll start another pull request and fork later today and fix those issues.